projects/004 - Finn/template-sql/covid-information.template.sql

```diff
@@ -23,22 +24,13 @@ SET @StartDate = '2020-02-01';
 -- OUTPUTS: #Patients
 
 --> EXECUTE query-patients-covid-tests.sql start-date:2020-02-01
--- OUTPUTS: #CovidPositiveTests, #CovidNegativeTests
-
--- Get all patients in the study cohort with a positive covid test and the date they tested positive.
--- Grain: multiple dates per patient, De-duped: Assume that a patient can have only one positive test per day. 
-IF OBJECT_ID('tempdb..#AllCohortCovidPositivePatients') IS NOT NULL DROP TABLE #AllCohortCovidPositivePatients;
-SELECT FK_Patient_Link_ID, CovidTestDate
-INTO #AllCohortCovidPositivePatients
-FROM #CovidPositiveTests
-WHERE FK_Patient_Link_ID IN (Select FK_Patient_Link_ID from  #Patients);
+-- OUTPUTS: #AllCovidTests
 
--- Get all patients in the study cohort with a Negative covid test and the date they tested Negative.
--- Grain: multiple dates per patient, De-duped: Assume that a patient can have only one Negative test per day. 
-IF OBJECT_ID('tempdb..#AllCohortCovidNegativePatients') IS NOT NULL DROP TABLE #AllCohortCovidNegativePatients;
-SELECT FK_Patient_Link_ID, CovidTestDate
-INTO #AllCohortCovidNegativePatients
-FROM #CovidNegativeTests
+-- Get all tests (positive, negative, excluded, assessed, and suspected), the test date and clinical code for the cohort patients
+IF OBJECT_ID('tempdb..#AllCohortCovidTests') IS NOT NULL DROP TABLE #AllCohortCovidTests;
```

Comment:
Is there a reason for basically copying this temp table into another one? Maybe there is, but I feel like I'm missing something.

Reply:
Not sure what you mean by this. The shared query 'query-patients-covid-tests' returns all patients with a recorded test and in the 'covid-information.template' (lines 30 and  34) get the tests related with the cohort.